File: articles/algorithmic-harm-error-and-institutional-responsibility/java/HarmRiskScore.java
public class HarmRiskScore {
    public static void main(String[] args) {
        double errorLikelihood = 0.34;
        double severity = 0.92;
        double exposure = 0.78;
        double contestability = 0.42;
        double harmRisk = errorLikelihood * severity * exposure * (1.0 - contestability);
        System.out.printf("harm_risk_score=%.4f%n", harmRisk);
    }
}

File: articles/algorithmic-harm-error-and-institutional-responsibility/fortran/harm_risk_score.f90
program harm_risk_score
  implicit none
  real :: error_likelihood, severity, exposure, contestability, harm_risk
  error_likelihood = 0.34
  severity = 0.92
  exposure = 0.78
  contestability = 0.42
  harm_risk = error_likelihood * severity * exposure * (1.0 - contestability)
  print *, "harm_risk_score=", harm_risk
end program harm_risk_score
